backend-api/.env

```diff
@@ -0,0 +1,3 @@
+CONNECTIONSTRING=mongodb+srv://mainUser:EJx3JFf4zh22F4kE@cluster0.qvt0x.mongodb.net/Fitigai?retryWrites=true&w=majority
```

Comment:
*This comment is for all the `backend-api` folder*

This repo is only for the frontend project, so I recommend you to add it into your `.gitignore` file so it doesn't pollute the frontend project